cumulus/xcm/xcm-emulator/src/lib.rs

```diff
@@ -578,7 +592,8 @@ macro_rules! decl_test_parachains {
 				},
 				pallets = {
 					$($pallet_name:ident: $pallet_path:path,)*
-				}
+				},
+				hooks = $on_hooks:ty,
```

Comment:
Don't we want to have a similar to `pallets` above way to set the `hooks`?
In most cases we do not need a custom with `OnHooks` trait, but we just want to list for which pallets it should run `on_initialize ` and `on_finalize`.
What you think? @0xmovses @NachoPal

Reply:
I am not sure that something like:
rust```
  hooks = {
	  $($pallet_name:ident: $pallet_path:path,)*
  }
```
could be possible to be passed as argument of `__impl_test_ext_for_parachain/relay()`

Having the `OnHooks` trait also gives more flexibility to the dev in case he wants to execute some custom code during `on_initialize/on_finalize()`